src/components/Footer/index.js

```diff
@@ -1,25 +1,48 @@
 import React from 'react';
 import {
-  Row, Column, Heading, Link, Paragraph, themeColor,
+  Row,
+  Column,
+  Heading,
+  Link,
+  themeColor,
+  themeSpacing,
 } from '@datapunt/asc-ui';
 import styled from 'styled-components';
 
 const Disclaimer = styled.div`
   background-color: ${themeColor('tint', 'level5')};
-  padding-top: 20px;
-  padding-bottom: 20px;
+  padding-top: 28px;
+  padding-bottom: 28px;
 
   * {
     color: ${themeColor('bright', 'main')};
   }
 `;
 
+const StyledLink = styled(Link)`
+  font-size: 16px;
+  font-family: Avenir Next LT W01 Demi, arial, sans-serif;
+
+  span {
+    align-self: center;
+  }
+`;
+
 const Privacy = styled.div`
   background: ${themeColor('bright', 'main')};
   padding-top: 10px;
   padding-bottom: 10px;
 `;
 
+const StyledHeading = styled(Heading)`
+  margin: 0 0 ${themeSpacing(3)} 0;
+`;
+
+const StyledText = styled.span`
+  font-size: 16px;
```

Comment:
Instead of doing this in various component, can't we set the base font size instead?

Reply:
gedaan